### PR TITLE
main/varnish: upgrade to 6.0.1

### PR DESCRIPTION
**Pull request by xpecex**
https://github.com/varnishcache/varnish-cache/blob/6.0/doc/changes.rst#varnish-cache-601-2018-08-29

### CODE REVIEW

File: main/varnish/APKBUILD [hunk 1]
```diff
@@ -20,7 +21,6 @@ pkggroups="varnish"
 source="http://varnish-cache.org/_downloads/$pkgname-$pkgver.tgz
 	musl-include-vpf.patch
 	musl-include-vsb.patch
-	musl-fix-stack-overflow.patch
```
Comment (andypost) [inline main/varnish/APKBUILD]: I need to remove the file as well

Comment (ncopa) [reply main/varnish/APKBUILD]: yes. `git rm musl-fix-stack-overflow.patch && git commit --amend` and then `git push --force`

**Comment by ncopa** [comment]
fixed it for you. thanks!

**Comment by ncopa** [comment]
fixed with ac1767b7571304789f0a6898993b68a0ebd49d37

**Pull request closed without merging**